docs/pages/extensibility/CustomTasks.md

```diff
@@ -1,12 +1,19 @@
 # Custom UI5 Builder Tasks
 
-The UI5 Build Extensibility enables you to enhance the build process of any UI5 project. In addition to the [standard task list](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.tasks.html), custom tasks can be created.
+The UI5 Build Extensibility enables you to enhance the build process of any UI5 project. In addition to the [standard tasks](../Builder.md#standard-tasks), custom tasks can be created.
+
+The UI5 community already created many custom tasks which you can integrate into your project. They are often prefixed as `ui5-task-` to make them easily searchable in the [npm registry](https://www.npmjs.com/search?q=ui5-task-).
+
+Please note that custom tasks from third parties can not only modify your project but also execute arbitrary code on your system. In fact, this is the case for all npm packages you install. Always act with the according care and follow best practices.
 
 ## Configuration
 
-You can configure your build process with additional build task. The custom tasks can be defined in the project [configuration](https://sap.github.io/ui5-tooling/pages/Configuration/) within the `ui5.yaml` file.
+You can configure your build process with additional build task. These custom tasks are defined in the project [configuration](../Configuration.md).
+
+To hook your custom tasks into the different build phase of a project, they reference other tasks to be executed before or after. This can be a [standard task](../Builder.md#standard-tasks) or another custom task. Note that a custom task will only be executed if the referenced task is executed (i.e. is not disabled).
```

Comment:
```suggestion
To hook your custom tasks into the different build phases of a project, they need to reference other tasks to be executed before or after. This can be a [standard task](../Builder.md#standard-tasks) or another custom task. Note that a custom task will only be executed if the referenced task is executed (i.e. is not disabled).
```